client/components/game/home/quaterfinal.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+
+var eventList = [{
+  advicer:"You finished the game in 10 minutes"
+},
+  {
+    advicer:"Your goal has been met"
+  }, 
+  {
+    advicer:"You won the game"
+  }
+];
+export default React.createClass({
+  eventList,
+  render() {
+    return (
+        <div>
+          <h1> Soda can production is increasing by 10%</h1>
```

Comment:
You should be referencing `this.props.event` and assuming that it has been passed in.

This component should not have any data itself.